civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -3,9 +3,76 @@
 
 /* Welcome page */
 .page-welcome-page {
-  .pane-user-login {
-    max-width: 438px;
-    margin: 0 auto;
-    width: 100%;
-  }
-}
\ No newline at end of file
+    .pane-user-login {
+        max-width: 438px;
+        margin: 0 auto;
+        width: 100%;
+    }
+}
+
+.page-reports {
+    .content {
+        padding: 0;
+        background-color: transparent;
+    }
+
+    .chr_panel__footer {
```

Comment:
I've removed the .chr_panel__footer styles, now it's using the .chr_panel classes (.chr_panel,  .chr_panel--no-padding, .pane-content).